examples/fluids/qfunctions/newtonwave.h

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+/// @file
+/// Thermodynamic wave propogation for testing freestream/non-reflecting
+/// boundary conditions. Proposed in Mengaldo et. al. 2014
+
+#include <ceed.h>
+#include <math.h>
+#include "utils.h"
+#include "newtonian_state.h"
+
+typedef struct NewtonWaveContext_ *NewtonWaveContext;
+struct NewtonWaveContext_ {
+  CeedScalar epicenter[2];     // Location of the perturbation
+  CeedScalar mean_velocity[2]; // Background advective velocity DELETE IF UNUSED
```

Comment:
Comment?

Reply:
Ah, yep. Thanks. Used to have a vim plugin that would highlight stuff like that (`DELETE`, `TODO`, etc.)